introduction.md

```diff
@@ -0,0 +1,2 @@
+test: this line has to be longer so I've made it longer
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.